acceptance/tests/dat_reader.rs

```diff
@@ -44,8 +44,6 @@ fn reader_test(path: &Path) -> datatest_stable::Result<()> {
     Ok(())
 }
 
-datatest_stable::harness!(
-    reader_test,
-    "tests/dat/out/reader_tests/generated/",
-    r"test_case_info\.json"
-);
+datatest_stable::harness! {
```

Comment:
does the new version of dat include a test of the functionality you've changed here?